### PR TITLE
Feat 645 subcategories

### PR DESCRIPTION
**Pull request by adamrocker**
Show subcategories in the keycodes area for users to find a key easily.

スクリーンショット 2021-10-04 17 01 12

In case of the search reslults, the subcategories are also shown.
スクリーンショット 2021-10-04 17 08 10

### CODE REVIEW

File: src/components/configure/keycodes/Keycodes.scss [hunk 1]
```diff
@@ -6,9 +6,51 @@
...
   max-width: 1280px;
   margin: 0 auto;
+  .sub-category-group {
+    display: flex;
+    flex-direction: row;
+    &:hover {
+      .keycodekey {
+        background-color: $primary-pale;
+        border-color: white;
+        transition: background-color 0.5s;
```
Comment (yoichiro) [inline src/components/configure/keycodes/Keycodes.scss:20]:
```suggestion
        border-color: $primary-light;
```

File: src/components/configure/keycodekey/KeycodeKey.scss [hunk 2]
```diff
@@ -13,6 +13,9 @@
   margin-top: 8px;
   z-index: 1;
   position: relative;
+  &:hover {
+    background-color: $primary-light !important;
```
Comment (yoichiro) [inline src/components/configure/keycodekey/KeycodeKey.scss:17]:
```suggestion
    background-color: $primary-pale !important;
```

**Review by yoichiro — CHANGES_REQUESTED** [review]
@adamrocker Thank you for creating this pull request. I posted two suggestions, but other diffs are good to me.

**Comment by adamrocker** [comment]
> @adamrocker Thank you for creating this pull request. I posted two suggestions, but other diffs are good to me.

@yoichiro Thanks. I applied your all commits.

**Comment by yoichiro** [comment]
@adamrocker It seems that the testing was failed on CI. Could you fix it?

**Review by yoichiro — APPROVED** [review]
@adamrocker LGTM. Thanks!

**Pull request merged**